src/abagnale.js

```diff
@@ -117,6 +131,10 @@ class Abagnale {
     Forge an ID for a single refract structure and any children in-place.
   */
   forgeOne(path, refract) {
+    if (!refract) {
+      return refract;
```

Comment:
It always returns the modified refract. In this case, if the refract passed in is bad, e.g. `null` or `undefined` then we just return it back out as-is. It helps with cases where the refract might be malformed, for example a member with a key that is `null` instead of a refract element.